src/assets/js/app.js

```diff
@@ -904,20 +904,29 @@ $(document).ready(function(){
         //Toggle target tab
         $($(this).attr('href')).addClass('show active').siblings().removeClass('show active');
 
-        //Change of behavior when clicked in accesibility nav-tabs 
-        if($(this).attr('id') === "android-tab"){
-            $(document).find("div#android").addClass('show active').siblings().removeClass('show active');
-        } else if(($(this).attr('id') ===  "ios-tab")){
-            $(document).find("div#ios").addClass('show active').siblings().removeClass('show active')
-        }
-
         //Keep selected on refresh except in FAQ results
         if (!document.querySelector(".page-faq-results")) {
             if(window.location.href.includes("#")) window.location.href = window.location.href.split("#")[0]+=$(this).attr('href');           
             else window.location.href += $(this).attr('href');
         }
       });
 
+    $('.mobile-type').ready(function(e){
+        if(location.hash === "#ios" || location.hash === "#android"){
+            $(window).scrollTop($("#glossary").offset().top);
+        }
+    });
+    $(window).bind( 'hashchange', function(e) { 
+        console.log("he sido cambiado")
```

Comment:
If this is intended as a permanent change then the text should be in ingles!

Reply:
yes sorry I was making some testing, now i have pushed a new commit.
thank you, see you tomorrow!